assets/javascripts/discourse/controllers/admin-plugins-explorer.js.es6

```diff
@@ -26,6 +26,16 @@ export default Ember.Controller.extend({
   sortBy: ["last_run_at:desc"],
   sortedQueries: Em.computed.sort("model", "sortBy"),
 
+  searchedContent: function() {
```

Comment:
We should use `computed` along with ES6 function syntax here.

Reply:
Done, sorry!